vendors/nordic/boards/nrf52840-dk/ports/ble/iot_ble_hal_common_gap.c

```diff
@@ -1425,17 +1423,6 @@ void prvPmEventHandler( const pm_evt_t * event )
 
     switch( event->evt_id )
     {
-        case PM_EVT_BONDED_PEER_CONNECTED:
```

Comment:
What's the context for removing this event instead of using pxPairingStateChangedCb?

Reply:
Bonding is part of the pairing process itself, persisting the security parameters. Hence bond state is included in pairing callback  instead of a separate callback. Also this event handling is not needed as anyways pairingStateChangeCallback will be called after bonded peer connects.

Reply:
Okay thank you